week-1/Homework/mandatory/4-tax.js

```diff
@@ -2,40 +2,57 @@
   SALES TAX
   =========
   A business requires a program that calculates how much sales tax to charge
-  Sales tax is 20% of the price of the product
-*/
+  Sales tax is 20% of the price of the product 
 
-function calculateSalesTax() {}
+  Remember that the prices must include the sales tax (hint: you already wrote a function for this!)*/
+
+const TAX_RATE = 0.2;
+
+function calculateSalesTax(productPrice) {
+  productPrice = productPrice + productPrice * TAX_RATE;
+  return productPrice;
+}
```

Comment:
First here, you don't need for a const tax outside of the function you could have created a variable that holds the 0.2 tax. Second, the variable name and the parameter have the same name which can confuse you and other developers. Third, store each step in a variable so that it can make sense. Fix those points please.